Index: dwl/config.def.h
===================================================================
--- dwl.orig/config.def.h
+++ dwl/config.def.h
@@ -57,7 +57,9 @@ static const struct xkb_rule_names xkb_r
 	/* example:
 	.options = "ctrl:nocaps",
 	*/
-	.options = NULL,
+    .layout = "us,no",
+    .variant = "colemak, colemak",
+    .options = "grp:shifts_toggle",
 };
 
 static const int repeat_rate = 25;
@@ -107,7 +109,7 @@ LIBINPUT_CONFIG_TAP_MAP_LMR -- 1/2/3 fin
 static const enum libinput_config_tap_button_map button_map = LIBINPUT_CONFIG_TAP_MAP_LRM;
 
 /* If you want to use the windows key for MODKEY, use WLR_MODIFIER_LOGO */
-#define MODKEY WLR_MODIFIER_ALT
+#define MODKEY WLR_MODIFIER_LOGO
 
 #define TAGKEYS(KEY,SKEY,TAG) \
 	{ MODKEY,                    KEY,            view,            {.ui = 1 << TAG} }, \
@@ -119,7 +121,7 @@ static const enum libinput_config_tap_bu
 #define SHCMD(cmd) { .v = (const char*[]){ "/bin/sh", "-c", cmd, NULL } }
 
 /* commands */
-static const char *termcmd[] = { "foot", NULL };
+static const char *termcmd[] = { "xfce4-terminal", NULL };
 static const char *menucmd[] = { "wmenu-run", NULL };
 
 static const Key keys[] = {
