Index: dwl/config.def.h
===================================================================
--- dwl.orig/config.def.h
+++ dwl/config.def.h
@@ -119,7 +119,7 @@ static const enum libinput_config_tap_bu
 #define SHCMD(cmd) { .v = (const char*[]){ "/bin/sh", "-c", cmd, NULL } }
 
 /* commands */
-static const char *termcmd[] = { "foot", NULL };
+static const char *termcmd[] = { "xfce4-terminal", NULL };
 static const char *menucmd[] = { "wmenu-run", NULL };
 
 static const Key keys[] = {
